ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/directionsquestionnaire/CCDDirectionsQuestionnaire.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.cmc.ccd.domain.directionsquestionnaire;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDAddress;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Builder
+@Value
+public class CCDDirectionsQuestionnaire {
+
+    private String languageInterpreted;
+    private String signLanguageInterpreted;
+    private CCDYesNoOption hearingLoop;
+    private CCDYesNoOption disabledAccess;
+    private String otherSupportRequired;
+
+    private String hearingLocation;
+    private String hearingLocationSlug;
+    private CCDCourtLocationOption hearingLocationOption;
+    private CCDAddress hearingCourtAddress;
+    private String exceptionalCircumstancesReason;
+
+    private CCDYesNoOption selfWitness;
+    private Integer howManyOtherWitness;
```

Comment:
is sancta fine with this? Shouldn't it  be noOfOtherWitness?

Reply:
Would also prefer `numberOfOtherWitnesses`

Reply:
changed to `numberOfOtherWitnesses` and updated ccd-domain

Reply:
changed to `numberOfOtherWitnesses` and update ccd domain